server/tsuserver.py

```diff
@@ -455,6 +463,16 @@ def send_arup(self, client, args):
 
         client.send_command('ARUP', *args)
 
+    def send_discord_chat(self, name, message, hub_id=0, area_id=0):
+        area = self.hub_manager.get_hub_by_id(hub_id).get_area_by_id(area_id)
+        cid = self.get_char_id_by_name(self.config['bridgebot']['character'])
+        message = remove_URL(message)
+        message = message.replace('}', '\\}').replace('{', '\\{').replace('`', '\\`').replace('|', '\\|').replace('~', '\\~').replace('º', '\\º').replace('№', '\\№').replace('√', '\\√').replace('\\s', '').replace('\\f', '')
+        message = self.config['bridgebot']['prefix'] + message
+        if len(name) > 14:
+            name = name[:14].rstrip() + '.'
+        area.send_ic(None, '1', 0, self.config['bridgebot']['character'], self.config['bridgebot']['emote'], message, self.config['bridgebot']['pos'], "", 0, cid, 0, 0, [0], 0, 0, 0, name, -1, "", "", 0, 0, 0, 0, "0", 0, "", "", "", 0, "")
+
     def refresh(self):
```

Comment:
Note that in some circumstances your bot token can be revoked automatically. You might want to come up with a mechanism to hot-restart the bot with a new token.